src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -480,8 +480,12 @@ components:
       properties:
         navigation:
           $ref: "#/components/schemas/Navigation"
-        clientApplication:
-          $ref: "#/components/schemas/ClientApplication"
```

Comment:
dont we need to remove the `ClientApplication` schema??

Reply:
no we have it on the generate url